app/views/play/CampaignView.coffee

```diff
@@ -273,6 +275,54 @@ module.exports = class CampaignView extends RootView
     @applyCampaignStyles()
     @testParticles()
 
+  onShiftS: (e) ->
+    @generateCompletionRates() if @editorMode
+
+  generateCompletionRates: ->
+    return unless me.isAdmin()
+    startDay = utils.getUTCDay -14
+    startDayDashed = "#{startDay[0..3]}-#{startDay[4..5]}-#{startDay[6..7]}"
+    endDay = utils.getUTCDay -1
+    endDayDashed = "#{endDay[0..3]}-#{endDay[4..5]}-#{endDay[6..7]}"
+    $(".map-background").css('background-image','url("")')
+    for level in @campaign?.renderedLevels ? []
+      $("div[data-level-slug=#{level.slug}] .level-kind").text("Loading...")
+      do (level) =>
+        console.log(level.slug)
+        request = @supermodel.addRequestResource 'level_completions', {
```

Comment:
Put this in its own `onLevelCompletionsLoaded` method instead of leaving it nested?